vitest.setup.ts

```diff
@@ -134,29 +145,78 @@ Note: test files can also be named `test-<browser target>.` instead of `test.` t
 */
 
 function parseTest(filename?: string) {
```

Comment:
I'll rename loader to loaderKind is the name of a loader isn't an actual loader.